tests/TestCase/Routing/Route/RouteTest.php

```diff
@@ -39,16 +39,24 @@ public function setUp(): void
         Configure::write('Routing', ['prefixes' => []]);
     }
 
+    public function testDeprecationWarning()
```

Comment:
```suggestion
    public function testDeprecatedPlaceholders()
```

Reply:
Don't be so pedantic :stuck_out_tongue:.